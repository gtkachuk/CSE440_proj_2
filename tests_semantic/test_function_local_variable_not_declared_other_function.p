program testFunctionLocalVariableNotDeclaredOtherFunction;


class testFunctionLocalVariableNotDeclaredOtherFunction

BEGIN
   VAR otherVar	: integer;
      

FUNCTION testFunctionLocalVariableNotDeclaredOtherFunction(value : integer): integer;
VAR
   someVar : integer;
BEGIN
   someVar := 6;
   testFunctionLocalVariableNotDeclaredOtherFunction := 2
END;

FUNCTION xxx;
BEGIN
   someVar := 5
END

END
.
