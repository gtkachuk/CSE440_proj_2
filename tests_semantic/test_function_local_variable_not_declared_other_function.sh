#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_function_local_variable_not_declared_other_function.p
