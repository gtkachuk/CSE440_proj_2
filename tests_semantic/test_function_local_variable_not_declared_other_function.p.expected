ERROR:21:Variable "someVar" not declared
Errors detected. Exiting.
